doc/nrf/app_dev/device_guides/nrf53/building_nrf53.rst

```diff
@@ -113,22 +125,26 @@ To program the nRF5340 DK from the command line, use either west (which uses nrf
 
          .. group-tab:: west
 
-            Enter the following command to program multi-image builds for different cores::
+            Enter the following command to program multi-image builds for different cores:
+
+            .. code-block:: console
 
-             west flash
+               west flash
 
-			   .. note::
-               The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
+			      .. note::
+                 The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
```

Comment:
```suggestion
            .. note::
               The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
```
Needs to be inline with code-block to render correctly.

Reply:
I don't see this issue on my machine, and the tabulation settings are ok. I did notice that the change that went in the previous push was different than on my machine, so tried to address this now.

Reply:
Still rendering wrong: 
![image](https://github.com/user-attachments/assets/07c8b291-8e6a-4994-9b0c-f5302efd8369)
I see a couple of tabs instead of spaces on Lines 134 and 135.